geom_mgr/GeomManager_smod.F90

```diff
@@ -292,7 +292,7 @@ module function make_mapl_geom_from_spec(this, spec, rc) result(mapl_geom)
       geom = factory%make_geom(spec, _RC)
       file_metadata = factory%make_file_metadata(spec, _RC)
       gridded_dims = factory%make_gridded_dims(spec, _RC)
-      mapl_geom = MaplGeom(spec, geom, file_metadata, gridded_dims)
+      mapl_geom = MaplGeom(spec, geom, factory, file_metadata, gridded_dims)
```

Comment:
OK - the list is getting long.   We'll have to watch for ways to combine things into related data types.